src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  padding: 10px;
+  width: 7 * ($width-day + $gap-day);
+  gap: $gap-day;
+
+  .calendar__day {
+    display: flex;
+    box-sizing: border-box;
+    position: relative;
+    width: $width-day;
+    height: 100px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+
+    border: 1px solid #000;
+    background-color: #eee;
```

don't forget to create variables for all colors
```suggestion
    background-color: ...;
```